packages/adena-extension/src/components/edint-network/edit-network/edit-network.tsx

```diff
@@ -0,0 +1,100 @@
+import React, { useCallback, useMemo } from 'react';
+import { EditNetworkWrapper } from './edit-network.styles';
+import SubHeader from '@components/common/sub-header/sub-header';
+import LeftArrowIcon from '@assets/arrowL-left.svg';
+import CustomNetworkInput from '@components/common/custom-network-form/custom-network-input';
+import BottomFixedButtonGroup from '@components/buttons/bottom-fixed-button-group';
+import RemoveNetworkButton from '../remove-network-button/remove-network-button';
+
+export interface EditNetworkProps {
+  name: string;
+  rpcUrl: string
+  rpcUrlError?: string;
```

Comment:
is this only one with optional?

Reply:
This is optional because it is not certain that an error message will be sent.
After validation, it contains the error contents of `rpcUrlError`.